functests/sctp/sctp.go

```diff
@@ -212,3 +316,13 @@ func jobForNode(name, node, app string, cmd []string, args []string) *k8sv1.Pod
 
 	return &job
 }
+
+func beforeAll(fn func()) {
```

Comment:
beforeAll was moved here https://github.com/openshift-kni/cnf-features-deploy/blob/edffd2d01e128a20d690f2710a2732d9bf455faf/functests/utils/execute/ginkgo.go#L7

Please use that